hrjobcontract/js/src/job-contract/services/utils.js

```diff
@@ -118,7 +118,8 @@ define([
                 var deffered = $q.defer();
 
                 API.resource('HRPayScale','get',{
-                    sequential: 1
+                    sequential: 1,
+                    is_active: 1
```

Comment:
hmmmm.... taking a second look at this I agree it's not easy to figure out what to test here. Maybe you can mock the API object and check if the gets called with the is_active parameter?
 Let's see what Alessandro thinks about this